packages/glimmer-runtime/tests/initial-render-test.ts

```diff
@@ -806,105 +785,8 @@ test("Case-sensitive tag has capitalization preserved", function() {
-// });
-
-// QUnit.skip("nested unsafe morph uses the correct namespace", function() {
-//   let template = compile('<svg>{{{foo}}}</svg><div></div>');
-//   let fragment = render(template, { foo: '<path></path>' }).fragment;
-
-//   equal(fragment.childNodes[0].childNodes[0].namespaceURI, svgNamespace,
-//         'element from unsafe morph has correct namespace');
-// });
-
-// QUnit.skip("svg can take some hydration", function() {
-//   let template = compile('<div><svg>{{name}}</svg></div>');
-
-//   let fragment = render(template, { name: 'Milly' }).fragment;
-//   equal(
-//     fragment.firstChild.childNodes[0].namespaceURI, svgNamespace,
-//     "svg namespace inside a block is present" );
-//   equalTokens( fragment.firstChild, '<div><svg>Milly</svg></div>',
-//              "html is valid" );
-// });
-
-// QUnit.skip("root svg can take some hydration", function() {
-//   let template = compile('<svg>{{name}}</svg>');
-//   let fragment = render(template, { name: 'Milly' }, env).fragment;
-//   let svgNode = fragment.firstChild;
-
-//   equal(
-//     svgNode.namespaceURI, svgNamespace,
-//     "svg namespace inside a block is present" );
-//   equalTokens( svgNode, '<svg>Milly</svg>',
-//              "html is valid" );
-// });
-
-// QUnit.skip("Block helper allows interior namespace", function() {
-//   let isTrue = true;
-
-//   env.registerHelper('testing', function(params, hash, blocks) {
-//     if (isTrue) {
-//       return blocks.template.yield();
-//     } else {
-//       return blocks.inverse.yield();
-//     }
-//   });
-
-//   let template = compile('{{#testing}}<svg></svg>{{else}}<div><svg></svg></div>{{/testing}}');
-
-//   let fragment = render(template, { isTrue: true }, env, { contextualElement: document.body }).fragment;
-//   equal(
-//     firstChild(fragment).namespaceURI, svgNamespace,
-//     "svg namespace inside a block is present" );
-
-//   isTrue = false;
-//   fragment = render(template, { isTrue: false }, env, { contextualElement: document.body }).fragment;
-//   equal(
-//     firstChild(fragment).namespaceURI, xhtmlNamespace,
-//     "inverse block path has a normal namespace");
-//   equal(
-//     firstChild(fragment).firstChild.namespaceURI, svgNamespace,
-//     "svg namespace inside an element inside a block is present" );
-// });
-
-// QUnit.skip("Block helper allows namespace to bleed through", function() {
-//   registerYieldingHelper('testing');
-
-//   let template = compile('<div><svg>{{#testing}}<circle />{{/testing}}</svg></div>');
-
-//   let fragment = render(template, { isTrue: true }, env).fragment;
-//   let svgNode = fragment.firstChild.firstChild;
-//   equal( svgNode.namespaceURI, svgNamespace,
-//          "svg tag has an svg namespace" );
-//   equal( svgNode.childNodes[0].namespaceURI, svgNamespace,
-//          "circle tag inside block inside svg has an svg namespace" );
-// });
-
-// QUnit.skip("Block helper with root svg allows namespace to bleed through", function() {
```

Comment:
Was previously ported to `updating-test.ts` as `HTML namespace from root element is continued to child templates`